docs/source/installation/migration/1-migrating-project.rst

```diff
@@ -147,6 +147,8 @@ before importing data from the old environment.
 4. Import the data to the new environment
 -----------------------------------------
 
+* Ensure you are running the following steps from a release created using the CommCare version that you should
+  have been provided.
```

Comment:
Totally up for continuing to make it clear that celery needs to be stopped, but it's tricky since it isn't as simple as stopping celery prior to running `load_domain_data`. If celery was running at any point after resetting postgres, it is likely the case that the db has been "corrupted" and needs to be reset again.

Reply:
That is tricky. So a deploy will restart Celery, but we need to run a deploy after the environment is reset. Hmmm.

Reply:
Well I think the framing should be we need to run a deploy before resetting the environment (I think I made that change here too), since the "current" release also impacts what state the database in migrated to when rebuilding the environment.

Reply:
Yeah on second thought, I like `downtime` for that reason in particular, and updated the instructions to select the kill option when prompted in b7b46e6231032bed549b5a9708e1f2db16454516